README.md

```diff
@@ -43,9 +51,9 @@ In all projects there needs to be a reliable method of source/version control. [
 ![Branching Strategy Overview](./assets/branching.png)
 
 **Branches:**
-- `master`/`release`: This is the live or production version of the code. The single source of truth for the most up-to-date code.
-- `dev`/`staging`: This is the staging version of the code. Testing of new features should be done here prior to a release to `master`/`release` branch.
-- `feature`: This branch should be where development takes place. Branch can be for individual developers or issues/bugs/features. Sometimes if the feature is big there will have to be subbranches merging into this. Naming isn't as important as long as the `pull request` to `dev`/`staging` includes the appropriate ticket number.
+- `master`/`release`/`prod`: This is the live or production version of the code. The single source of truth for the most up-to-date code.
+- `dev`/`staging`/`full`/`partial`: This is the staging version of the code. Testing of new features should be done here prior to a release to `master`/`release`/`prod` branch.
```

Comment:
What do you think about changing this to just `dev`/`staging`/`full`? Just so we differentiate between a `staging` environment and a `development` environment.

Reply:
I'd think Staging is probably partial or full and Dev is probably partial or dev

Reply:
Okay, I'll reword to make this more clear